src/components/Book.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import capitalize from '../helpers/capitalize';
+import './Book.css';
+import { removeBook } from '../actions/index';
```

Comment:
- [ ] Since `Book` component is a presentational component, it shouldn't connect to redux store. `BookList` is a container component and It should connects `REMOVE_BOOK` action and then passes it to `Book`.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone3/ImplementsRemoveBook.png)